contributors/guide/README.md

```diff
@@ -251,6 +253,14 @@ Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
 Helping to manage or triage these open issues can be a great contribution and a great opportunity to learn about the various areas of the project. Triaging is the word we use to describe the process of adding multiple types of descriptive labels to GitHub issues, in order to speed up routing issues to the right folks.
 Refer to the [Issue Triage Guidelines](/contributors/guide/issue-triage.md) for more information.
 
+# Kubernetes Contributor Playground
+
+If you are looking for a safe place, where you can familiarize yourself with (some of) the Kubernetes Project's review and pull request processes. Then the [Kubernetes Contributor Playground](https://github.com/kubernetes-sigs/contributor-playground/) is the right place for you.
```

Comment:
nit: maybe `Then..` doesn't need to be a new sentence?

Reply:
you are right, just updated it.